app/js/appRouter.js

```diff
@@ -8,7 +8,7 @@ define(function(require) {
 	var Marionette = require('marionette');
 
 	var routes = {
-		'gameWindow(/)': 'gameWindow',
+		'gameWindow(/)(:id)': 'gameWindow',
```

Comment:
Just got an idea that maybe `gameWIndow(/)(/:id)` could work better :smiley:. But it can also fire on double-slash **gameWindow//c278** I guess :laughing:.